### ISSUE TITLE
How to integrate Reactstrap into mern-starter

### ISSUE THREAD

**Issue by chung2014** [Hashnode/mern-starter#406]
I would like to use `reactstrap` with this mern-starter. But I find that I cannot start the project with `npm start` properly after I add an import statment in `client/modules/App/App.js`
e.g
`import 'bootstrap/dist/css/bootstrap.css';`

**Comment by theevilhead** [comment]
Hey @chung2014,
I think the error you are getting is related to webpack loader.
It just needs to be loaded no need to complie it. 

Add the below to webpack.config.babel.js just like in the image below for now and use minified version `import 'bootstrap/dist/css/bootstrap.min.css'`;

If you want to customize bootstrap I think its better to follow bootstrap guide to importing styles/compile sass
https://getbootstrap.com/docs/4.0/getting-started/webpack/
```
{
        test: /\.min.css$/,
        use: ['style-loader', 'css-loader']
},
```
<img width="1280" alt="screen shot 2018-12-04 at 3 34 57 pm" src="https://user-images.githubusercontent.com/12823517/49434440-39d52000-f7da-11e8-9d9a-1e7a08079783.png">

**Comment by chung2014** [comment]
@theevilhead Yes, working with webpack settings is complex. And your suggest works. After adding the snippet into webpack.config.babel.js, I can run `npm run start` without error. How about if I run `npm run build` or `npm run build:server`, do I need to change my webpack config files for those environment as well?

**Comment by chung2014** [comment]
@theevilhead After I check the webpack config files, all `webpack.config.(dev|prod|server).js` has the code snippet you suggested, why is it the case that only `webpack.config.babel.js` doesn't contains it. Should all 4 config file be same?

**Comment by theevilhead** [comment]
Hey @chung2014  sorry for the delay in reply. Glad the suggestion works and I cannot be of more help here because my involvement with project has started recently, so I think the original contributors might have missed it but I dont know the exact intent though.
